x-pack/plugin/core/template-resources/src/main/resources/profiling/component-template/profiling-events.json

```diff
@@ -76,6 +76,9 @@
         },
         "tags": {
           "type": "keyword"
+        },
+        "service.name": {
+          "type": "keyword"
```

Comment:
The APM trace index restricts this with a `"ignore_above": 1024`. Should we mirror this or would you rather keep it consistent with our other fields that are equally unrestricted despite all being user-controlled?